### PR TITLE
Avoid copying data just to compress them when it is not necessary.

### PR DESCRIPTION
**Pull request by baldersheim**
@bratseth or @havardpe PR

### CODE REVIEW

**Comment by baldersheim** [comment]
Or @hmusum PR

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/rpc/SlimeClusterStateBundleCodec.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public EncodedClusterStateBundle encode(ClusterStateBundle stateBundle) {
         }
 
         byte[] serialized = BinaryFormat.encode(slime);
```
Comment (hmusum) [inline clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/rpc/SlimeClusterStateBundleCodec.java:50]:
```suggestion
```

Comment (hmusum) [reply clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/rpc/SlimeClusterStateBundleCodec.java:50]: Unused, can be removed

**Review by hmusum — APPROVED** [review]
LGTM

**Pull request merged**